### PR TITLE
GetBiometricTypeAsync

### PR DESCRIPTION
**Pull request by robertmaxted**
This PR is to allow for the retrieval of the hardware type on iOS and Android. @FreakyAli  Please pull and deploy on your test environment to ensure it works as expected. I am unable to test the sample application on my environment.

### CODE REVIEW

**Comment by robertmaxted** [comment]
@FreakyAli I am assuming I should not have forked from development. Please see my changes and add accordingly.. Regards, Rob.

**Comment by FreakyAli** [comment]
@robertmaxted Thanks for this, I will check everything out and merge accordingly :)

**Comment by robertmaxted** [comment]
@FreakyAli No problem, hope it all works as it should!

File: Plugin.Maui.Biometric/Plugin.Maui.Biometric/Shared/BiometricType.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+namespace Plugin.Maui.Biometric.Shared
```
Comment (FreakyAli) [inline Plugin.Maui.Biometric/Plugin.Maui.Biometric/Shared/BiometricType.cs:2]: @robertmaxted Can you please remove Shared from the namespace...

Comment (robertmaxted) [reply Plugin.Maui.Biometric/Plugin.Maui.Biometric/Shared/BiometricType.cs:2]: @FreakyAli I think that is due to it all being in develop, can you lift and shift all of the logic into a new branch based on the correct branch (master ? ).

Comment (robertmaxted) [reply Plugin.Maui.Biometric/Plugin.Maui.Biometric/Shared/BiometricType.cs]: done

**Comment by robertmaxted** [comment]
@FreakyAli can you take a look when you have a spare minute please mate?

**Comment by FreakyAli** [comment]
@robertmaxted  Hey man sorry i was a bit busy, I will check this over the weekend and see if i can make a release, hope you don't mind!

**Review by FreakyAli — APPROVED** [review]
Looks good for now, thanks!

**Comment by robertmaxted** [comment]
@FreakyAli awesome, cheers mate!

**Comment by FreakyAli** [comment]
@robertmaxted  Check this PR, Comment if you have any thoughts 
https://github.com/FreakyAli/Plugin.Maui.Biometric/pull/6

**Pull request merged**